content/geschaftsordnung.md

```diff
@@ -2,7 +2,7 @@
 title: "Geschäftsordnung"
 summary: "asdf"
 date: 2023-05-03T10:15:07+02:00
-draft: false
+draft: true
```

Comment:
Is there a reason for this? Setting this to true would result in this not being published on the Website

Reply:
resolved in "undo draft"